kitsune/community/templates/community/email/first_answer.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'email/base.html' %}
+
+{% block content %}
+    <p>
```

Comment:
We aren't consistent, unfortunately, but you're right, these should be two-space. I think we want to standardize on that.